Pythonwin/pywin/scintilla/bindings.py

```diff
@@ -6,7 +6,9 @@
 import win32con
 import win32ui
 
-from . import IDLEenvironment, keycodes
+from . import (  # nopycln: import # Injects fast_readline into the IDLE auto-indent extension
+    IDLEenvironment,
+)
```

Comment:
If injecting in the extension isn't needed when `IDLEenvironment` isn't used directly (it's imported in a couple other places), then this can be removed.